apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/responses/components/ResponsePage.tsx

```diff
@@ -152,6 +160,9 @@ const ResponsePage = ({
         hasMore={hasMore}
         deleteResponse={deleteResponse}
         updateResponse={updateResponse}
+        fetchingFirstPage={isFetchingFirstPage}
```

Comment:
why not naming it `isFetchingFirstPage` in the props, too?
that would be more consistent with our current naming.

Reply:
yup, true.
renaming it.